app/styles/partials/form.scss

```diff
@@ -18,3 +18,49 @@ label.required {
 input.preview {
   text-overflow: ellipsis !important;
 }
+
+.ui.aligned.segment.basic.forgot-password {
```

Comment:
form.scss is for classes that are common to _all_ or _most_ of the forms. Forgot password, reset password etc are not common for all.